pkg/receiver/gitproviderreceiver/internal/scraper/githubscraper/genqlient.graphql

```diff
@@ -105,6 +117,10 @@ query getBranchData(
   }
 }
 
+# This query currently costs 2 point per repo. REEVALUATE THIS ESTIMATE if you
+# change this query. And please update our docs if you do!
+# docs: pkg/receiver/gitproviderreceiver/README.md#rate-limiting
+# https://docs.github.com/en/graphql/overview/rate-limits-and-node-limits-for-the-graphql-api#predicting-the-point-value-of-a-query
 query getPullRequestData(
```

Comment:
This query costs 1 point per repo per page of pull requests. Each page contains 100 pull requests.